src/main/kotlin/datamaintain/Config.kt

```diff
@@ -30,6 +31,10 @@ class Config(val path: Path,
         return this
     }
 
+    infix fun isScriptBlacklisted(script: Script): Boolean {
```

Comment:
Not sure if making this function infix really is useful. Which one is clearer, ``config isScriptBlacklisted script`` or ``config.isScriptBlacklisted(script)``? I'm more used to the second one and don't see the use of the first one.